api/src/main/java/org/pmiops/workbench/actionaudit/bucket/BucketAuditQueryServiceImpl.java

```diff
@@ -0,0 +1,96 @@
+package org.pmiops.workbench.actionaudit.bucket;
+
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.QueryParameterValue;
+import com.google.cloud.bigquery.TableResult;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.StreamSupport;
+import javax.inject.Provider;
+import org.pmiops.workbench.api.BigQueryService;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.config.WorkbenchConfig.BucketAuditConfig;
+import org.pmiops.workbench.utils.FieldValues;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BucketAuditQueryServiceImpl implements BucketAuditQueryService {
+
+  private final Provider<WorkbenchConfig> workbenchConfigProvider;
+  private final BigQueryService bigQueryService;
+
```

Comment:
The query return the pet account, the sum of the file length, and the first and last timestamp of created files within this duration. As an extra information in the egress event